transport-native-epoll/src/main/java/io/netty/channel/epoll/Native.java

```diff
@@ -102,23 +102,23 @@ public static FileDescriptor newTimerFd() {
     public static native void eventFdWrite(int fd, long value);
     public static native void eventFdRead(int fd);
     static native void timerFdRead(int fd);
+    static native void timerFdSetTime(int fd, int sec, int nsec);
```

Comment:
This should have a `throws` as it may throw a `ClosedChannelException`

Reply:
However this is a runtime exception ... would you prefer a `IOException` using the `errno` like in `epollWait`?

Reply:
yeah I think this would be better (using NativeIOException)

Reply:
I just went with `IOException` for now ... avoids additional conditional checking and the string description of the error is already included in the `IoExecution` thrown by the JNI code.